File: pb6.m
function res=pb6()
tic
%
squaresOfSums=power(sum(1:100),2);
sos=zeros(1,10);
for i = 1:100
    sos(i) = power(i,2);
end
sumOfSquares= sum(sos);
res=squaresOfSums-sumOfSquares;
toc

File: pb6.py
def sumOfSquares(N):
	return (sum([x**2 for x in range(1,N+1)]))
def squareOfSums(N):
	return((sum([x for x in range(1,N+1)]))**2)


print squareOfSums(100)-sumOfSquares(100)
